rootfs/standard/var/pynode/device_info.py

```diff
@@ -1228,6 +1228,19 @@ def clear_mempool_cache():
     os.system("sync")
     os.system("systemctl restart mempool")
 
+#==================================
+# LNbits Functions
+#==================================
+def delete_lnbits_settings():
```

Comment:
I also don't think this would work. It looks like this launches a new container rather than running a command in the existing lnbits container.

Reply:
Have you tried it? It may work by launching a second container. Alternatively, you could exec into the running container to run the command, but I guess that would only clear the settings if lnbits was actively running.